pkg/kubelet/kubelet.go

```diff
@@ -2041,6 +2036,9 @@ func (kl *Kubelet) HandlePodRemoves(pods []*v1.Pod) {
 			kl.handleMirrorPod(pod, start)
 			continue
 		}
+		if _, ok := kl.podManager.GetMirrorPodByPod(pod); ok {
```

Comment:
mirror pods never get to this point due to the `continue` on 2036

Reply:
This call is for pod which has mirror pod.
That's why I place it here after the kubetypes.IsMirrorPod() call.